### DISCUSSION TITLE
renovate-config-validator fails to detect errors on extends

### DISCUSSION THREAD

**Discussion by Nithos** [renovatebot/renovate#15979] [Request Help]
During testing we noticed that the pipeline we run for renovate (self hosted GitLab) catches an error when there is a bad preset being used in the `extends` section of the configuration.

`  
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":rebaseStalePrs",
    ":pinVersions",
    "on Wednesday and Friday"
  ]
}
`

The above is caught correctly with the following error:
`"error": {
         "validationError": "Cannot find preset's package (on monday and wednesday)",
         "message": "config-validation",`

However this does not seem to be caught when using  `renovate-config-validator`
`
npx --package renovate renovate-config-validator
`

Returns
`INFO: Validating renovate.json5
 INFO: Config validated successfully`

It does catch any invalid properties properly however.

**Comment by rarkins** [comment]
Please create a bug report issue with reproduction repo of the above
